src/dump.ts

```diff
@@ -145,10 +147,57 @@ export interface ApiDump {
     Version: number,
 }
 
-let apiDumpPromise = (async () => {
-    return JSON.parse(await request(API_DUMP).catch((err) => {
+/*
+    Code borrowed from https://github.com/evaera/vscode-roblox-api-explorer/blob/master/src/api.ts
+*/
+async function injectDescriptions(classes: ApiClass[]) {
```

Comment:
I'm not really a fan of functions like this that mutate over their properties. Can you make this treat `classes` as immutable and return a new version?